webapp/channels/src/components/drafts/draft_row.tsx

```diff
@@ -224,6 +254,13 @@ function DraftRow({
         };
     }, [item, dispatch, connectionId]);
 
+    const handleScheduledPostOnSend = useCallback(async () => {
+        isScheduledPostBeingSent.current = true;
+        const postDraft = scheduledPostToPostDraft(item as ScheduledPost);
+        handleOnSend(postDraft, undefined, {keepDraft: true});
+        return Promise.resolve({});
```

Comment:
We don't want to return anything in this case?
Also, any strong reason to make this async if we are not really waiting for anything?

Reply:
So `handleOnSend` doesn't return any errors, I'm using the after optimistic send function for managing the error.